source/YugiAcademy/YuGiOh/static/css/styles.css

```diff
@@ -7,6 +7,10 @@
   transform: translateX(calc((100% - 100vw) / 2));
 }
 
+#welcome-video-message{
+  -webkit-text-stroke: 0.5px black;
```

Comment:
Replace this with something not specific to a browser family.

Reply:
https://css-tricks.com/almanac/properties/t/text-stroke/